src/main/java/org/fhirvp/usecase/impl/builder/FHIRResourceBuilderUseCaseImpl.java

```diff
@@ -0,0 +1,178 @@
+package org.fhirvp.usecase.impl.builder;
+
+import com.ibm.fhir.model.resource.Basic;
+import com.ibm.fhir.model.resource.ImmunizationRecommendation;
+import com.ibm.fhir.model.type.*;
+import org.fhirvp.Constants;
+import org.fhirvp.model.ForecastStatus;
+import org.fhirvp.ports.impl.fhir.context.FHIRClientConfig;
+import org.fhirvp.usecase.FHIRResourceBuilderUseCase;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.lang.String;
+import java.time.LocalDate;
+import java.util.Calendar;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@ApplicationScoped
+public class FHIRResourceBuilderUseCaseImpl implements FHIRResourceBuilderUseCase {
+
+    @Inject
+    FHIRClientConfig fhirClientConfig;
```

Comment:
You don't need this, the FHIR server accepts relative URLs without the base URL.

Reply:
Addressed at a9b6f4d